concordia/admin.py

```diff
@@ -103,11 +103,12 @@ def admin_bulk_import_view(request):
                 import_url_blob = row["Import URLs"]
 
                 if not all((campaign_title, project_title, import_url_blob)):
-                    messages.add_message(
-                        request,
-                        messages.WARNING,
-                        f"Skipping row {idx}: at least one required field (Campaign, Project, Import URLs) is empty",
+                    warning_message = (
+                        f"Skipping row {idx}: at least one required field \
```

Comment:
Putting the line continuation in there puts all of the subsequent indentation into the string. I'd just split this into two implicitly-concatenated strings:

```suggestion
                        f"Skipping row {idx}: at least one required field" \
```

Reply:
Done